.github/workflows/pkg-go-release.yaml

```diff
@@ -0,0 +1,31 @@
+name: Language Go - Release
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
```

Comment:
This probably shouldn't run on merge to main, just tag push

Reply:
Artifact from before we talked about automatic releases. Removed.